packages/math-input/src/components/input/math-input.tsx

```diff
@@ -76,6 +77,7 @@ class MathInput extends React.Component<Props, State> {
 
     state: State = {
         focused: false,
```

Comment:
I ended up deciding that we needed to separate the state regarding the focus style and the actual focus of the input, in order to support tabbing to the math input and receiving focusStyling WITHOUT closing the keypad. 

Separating the logic allows us to (1) separately handle the focus styling, (2) show the custom keypad on input focus, (3) keep the custom keypad open if the user tabs into the keypad, and (4) maintain the blurOnClickOutside functionality. 

I'm unsure if I should update the focused state property to something like "inputActive". Totally open to suggestions on this!